masterlist.yaml

```
@@ -15906,24 +15924,26 @@ plugins:
     clean:
       - crc: 0xB4C1E773
         util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkTitleRemover.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+  - name: 'ClockworkSuperiorSorting.esp'
+    req:
+      - *PapyrusExtender
+      - *SKSE
     clean:
-      - crc: 0x913191E9
-        util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkMausoleumDummyDoorFix.esp'
+      - crc: 0x9E78E8D9
+        util: 'SSEEdit v4.0.4'
+  - name: 'ClockworkTitleRemover.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
-    clean:
-      - crc: 0xCAE2B6AE
-        util: 'SSEEdit v4.0.3g'
+    dirty:
+      - <<: *quickClean
+        crc: 0xBF1FBE1F
+        udr: 3
```

Missing a `util`, & I get a different checksum.

```suggestion
    dirty:
      - <<: *quickClean
        crc: 0x913191E9
        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        udr: 3
```